### ISSUE TITLE
ImportError: cannot import name 'GroupCommand' from 'keepercommander.commands.base'

### ISSUE THREAD

**Issue by variable** [Keeper-Security/Commander#926]
Getting this error
```
>>> from keepercommander import commands
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jlin/virtualenvs/playground/lib/python3.8/site-packages/keepercommander/commands/__init__.py", line 12, in <module>
    from .base import register_commands, register_enterprise_commands, aliases, commands, command_info, \
  File "/Users/jlin/virtualenvs/playground/lib/python3.8/site-packages/keepercommander/commands/base.py", line 29, in <module>
    from .. import api, crypto, utils, vault
  File "/Users/jlin/virtualenvs/playground/lib/python3.8/site-packages/keepercommander/api.py", line 31, in <module>
    from . import constants, rest_api, loginv3, utils, crypto, vault
  File "/Users/jlin/virtualenvs/playground/lib/python3.8/site-packages/keepercommander/loginv3.py", line 25, in <module>
    from .breachwatch import BreachWatch
  File "/Users/jlin/virtualenvs/playground/lib/python3.8/site-packages/keepercommander/breachwatch.py", line 17, in <module>
    from .commands.breachwatch import BreachWatchResetCommand
  File "/Users/jlin/virtualenvs/playground/lib/python3.8/site-packages/keepercommander/commands/breachwatch.py", line 19, in <module>
    from .base import GroupCommand, Command, dump_report_data
ImportError: cannot import name 'GroupCommand' from partially initialized module 'keepercommander.commands.base' (most likely due to a circular import) (/Users/jlin/virtualenvs/playground/lib/python3.8/site-packages/keepercommander/commands/base.py)
```

version keepercommander==16.8.10

Seems like a circular import as suggested

**Comment by variable** [comment]
It looks like I first need to run `from keepercommander import api, vault, vault_extensions, record_management,generator` to avoid the circular import even though I don't need any of those.

**Comment by sk-keeper** [comment]
Thank you for letting us know about this circular import. It will be fixed in the next Commander release

**Comment by sk-keeper** [comment]
Fixed in 16.8.14
